src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -103,7 +103,9 @@ public CryptoProviderFactory(CryptoProviderFactory other)
         /// <summary>
         /// Gets the <see cref="CryptoProviderCache"/>
         /// </summary>
-        public CryptoProviderCache CryptoProviderCache { get; } = new InMemoryCryptoProviderCache();
+        /// Set was added for simplifying stress and perf tests.
```

Comment:
remarks section? notes for review?

Reply:
I believe Brent added this in. I'm interpreting this as notes for review; I don't think it's the kind of content that belongs in a remarks section....

Reply:
set is good for testing too, I say we keep it and remove the comment, thoughts?